packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleTableTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+using System.Text.Json;
+using Bunit;
+using Xunit;
+#nullable enable
+namespace NimbleBlazor.Tests.Unit.Components;
+
+/// <summary>
+/// Tests for <see cref="NimbleTable"/>
+/// </summary>
+public class NimbleTableTests
+{
+    internal class TableRecord
+    {
+        public TableRecord(string firstName, string lastName)
+        {
+            FirstName = firstName;
+            LastName = lastName;
+        }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+    }
+
+    internal class BadTableRecord
+    {
+        public BadTableRecord(string firstName, string lastName, BadTableRecord? parent = null)
+        {
+            FirstName = firstName;
+            LastName = lastName;
+            Parent = parent;
+        }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public BadTableRecord? Parent { get; set; }
+    }
+
+    [Fact]
+    public void NimbleTable_Rendered_HasTableMarkup()
```

Comment:
Can you think of a way to write a unit test that verifies that the rows are put in the table?

Reply:
That's what I figured, and I agree that we shouldn't hold this PR up for that.